docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
+## Redelegation of tokens
 
-* The validator needs to unjail itself after at least 600 seconds of being jailed. There is an unjail transaction that can be sent via the CLI from within the Validator account/node. 
-
-```shell
-cudos-noded tx slashing unjail --from mykey [flags]
-```
+A **staker/delegator** can choose to **undelegate** their tokens at any time. 
 
+A **staker/delegator** can **redelegate** their tokens from **Validator A** to **Validator B** without any penalties or extra waiting times. However, they need to wait for 21 days if they want to further redelegate any amount of tokens from Validator B.
 
-For each validator, jailing starts after signed_blocks_window + start_height many blocks.
-We have signed_blocks_window = 19200.
+To redelegate tokens from **Validator B** to **Validator C** there is a *21 day waiting period*.
 
-This means that the earliest a validator can be jailed is 19200 blocks after they join the network.
+A **validator** can only **self-delegate through `cudos-noded CLI`**
 
-Afterwards, they will get jailed as soon as they miss 17280 blocks in any interval of 19200 blocks.
+A **validator** can also add **additional stake** to it via the `cudos-noded CLI` to increase its voter power.
 
-If a validator signs more than one block at the same height, that's also an offence punishable by slashing 5% of their tokens.
+## Block Rewards 
 
-This could be the case if someone duplicates their validator and try to double their voting power.
+The more tokens staked to a **Validator node**, the higher the likelihood of it being chosen to propose and sign a block. 
 
-Slashed tokens are sent to the community pool.
+* **Block rewards** are calculated after a block is issued. 
 
+:::info Block rewards
 
-## Transactions 
+*Block rewards* =  *staking rewards* + *tx fees* 
+:::
 
-How to view the mempool? Or pending transactions 
-If you're a validator you can keep your mempool to yourself when it is your turn to propose a block then you can submit your own transactions. 
+* Staking rewards occur in a **single pool** and are distributed as follows:  
 
-If you sync your own node with other nodes then you can view pending transactions.
+    1. Validators are assigned rewards proportional to their 
+    
+    2. Users who have delegated tokens are required to **actively** claim those rewards by sending appropriate txs to the blockchain.
 
-* Tx fees are calculated based on the **gas_wanted value rather than gas_used**. Therefore users need to be aware of this and should not provide extra gas_wanted values “just in case”.
+    3. The **block proposer** gets a base bonus as 1% of the block rewards
+    * In addition, it gets (4*P)% percent of the block rewards where P=(total power of validators with included precommits / total bonded validator power)
+    * This means it gets an additional bonus between 2.67% and 4%
+    * This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.
```

Comment:
Unless this document is intented to be technically detailed, maybe this could be simplified as "The block proposer receives as a bonus between 3.67% and 5% of the total block rewards depending on the number of validators signing that block." Otherwise we would need to explain in more detail the terms in the formula, e.g. precommit.

Reply:
Done